### DISCUSSION TITLE
What does 'backcountry' answer imply? (Who may camp here?)

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#5712] [Q&A]
Prompted by a OSM forum [discussion](https://community.openstreetmap.org/t/backcountry-camping-but-no-tents-allowed/115348) on whether or not a backcountry area in which using a lean-to is the only authorised type of overnight stay is a tourism=camp_site, I have a question regarding the 'Who may camp here?'-quest:

There are three possible answers to this quest: Tents only, Caravan only or Both. However, there is also the 'Uh…' option 'Backcountry camping'. I am not sure how to interpret this as an answer to the quest? From the OSM forum discussion it seems that some (most?) assume that 'backcountry' means 'no access for motorised vehicles, but would it not still also need to be specified whether or not tents are allowed for the backcountry campsite?

**Comment by matkoniecz** [comment]
https://github.com/streetcomplete/StreetComplete/pull/4213

@mnalis

**Reply by ERYpTION** [reply]
OK, so the Uh-option 'backcountry' is not actually related to the allowed mode of camping. I still think having this Uh-option is a little confusing since it seems that the quest question could also be asked for a backcountry campsite. At least regarding tents.

**Reply by matkoniecz** [reply]
Note that: SC may be wrong about tags (create issue in such case), people participating in discussion may be wrong and so on. Oh, and OSM Wiki can be also wrong.

I ma not very familiar with this case but note that such discussion cannot redefine what tag means (no discussion outcome will change already tagged features to magically resurvey itself to match new meaning) but may result in discovery that actual tag use is different than people thought.

And there is fine line between "this is a common mistake" and "tag changed meaning over time".

**Reply by mnalis** [reply]
ref: #5714 - I'm going to answer there to avoid duplicate information

**Reply by ERYpTION** [reply]
I apologise in advance if I misunderstood the information given in the other thread. I will take the liberty of replying here anyway. To me it seems like two different issues.

My objection is about the 'Who may camp here?'-quest, where I do not think it makes sense to answer 'backcountry' (in the Uh-options), since I think the original question should be asked regardless of whether it is a 'backcountry' campsite or not. At least as far as tents are concerned although I guess for a backcountry campsite a lean-to should also be a possible answer.

**Reply by matkoniecz** [reply]
I guess that for some backcountry ones it may be hard or confusing to answer the main question? Though maybe leaving notes in such cases would also make sense?

**Reply by ERYpTION** [reply]
Yes, you could be right, I suppose a reason for not having this quest for backcountry campsites could be difficulty in determining the allowed usage. I don't know how common it is in other countries, but here in Denmark there is sometimes a sign saying so if you are not allowed to pitch a tent at a backcountry campsite (by a lean-to). I am not sure about signs forbidding caravans but since these are rarely able to drive all the way into the campsite it is probably not necessary.

**Comment by rhhsm** [comment]
From the forum discussion is appears that sleeping in a lean-to is also considered camping, so it might be added as an answer option to this quest. But I suppose the use case for this information is to know if you can camp there if you bring **your own** tent/caravan/camper van, and since a lean-to will already be there (and probably mapped as such), it might not be needed as an answer option (a note could always be created about it if necessary).

I don't understand why the 'Backcountry camping' is there as an 'Uh…' option. Backcountry sites are very likely to be either "tents only" or "neither tent, caravan or camper" (if it only allows sleeping in a lean-to) so this can be answered in the main quest dialogue.

Maybe the reply interface of the quest should be redesigned so that for each of the 3 options, a "yes" or a "no" box can be ticked (or left empty, if not known)?

**Reply by mnalis** [reply]
> From the forum discussion is appears that sleeping in a lean-to is also considered camping

That seems to be contentious.  [tourism=camp_site wiki](https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dcamp_site) specifically states that _"while camp sites might (or might not) include occasional [building](https://wiki.openstreetmap.org/wiki/Key:building)=[cabin](https://wiki.openstreetmap.org/wiki/Tag:building%3Dcabin) or other housing accommodation, those are neither required nor indicative of [tourism](https://wiki.openstreetmap.org/wiki/Key:tourism)=camp_site (which is defined by having pitches for placing tents and/or caravans/RVs)"_

So it should allow either tents or caravans (or both) to be considered `tourism=camp_site`. Some interpretations of `backcountry=yes` are already on the edge of the definition there (e.g. if there are no designated pitches), but if both tents **and** caravans are forbidden, then it is not `tourism=camp_site` (even if it may be [amenity](https://wiki.openstreetmap.org/wiki/Key:amenity)=[shelter](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dshelter) +  [shelter_type](https://wiki.openstreetmap.org/wiki/Key:shelter_type)=lean_to). IOW, not every place where it is not illegal to camp is `tourism=camp_site` (as noted in that community discussion too).

> I don't understand why the 'Backcountry camping' is there as an 'Uh…' option.

Perhaps that `Uh...` option should be removed completely, and `AddCampType` asked unconditionally on all `tourism = camp_site` (regardless of any `backcountry=*` tag)? It was not done originally, as it was not unambiguous definition what `backcountry=yes` represents, so it seemed as safest bet to just skip asking asking the type there, as it was more likely to confuse mappers than help. E.g. I was afraid that many of them would leave notes _"I'm not sure, there are no separate pitches,  as it is backcountry camping"_; and tagging as `backcountry=yes` seemed like preferred solution at the time.

> Maybe the reply interface of the quest should be redesigned so that for each of the 3 options, a "yes" or a "no" box can be ticked (or left empty, if not known)?

Note that this would likely fail [quest guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md), specifically: _"⚛️ Atomic quests: Per quest, only one thing should need to be answered by the user."_ 
Also, how would such "unknown" values be tagged, to satisfy _"🤔 No unanswerable quests: All generated quests need to be actually answerable (no false-positives). This means that **any answer** given by the user **must result in something being tagged**"_ ?

**Reply by rhhsm** [reply]
> That seems to be contentious.

It goes against my feeling too, but that's what the forum discussion tends towards. I think you should post your opinion there.

> Perhaps that `Uh...` option should be removed completely,

Yes, that's what I tried to suggest above.

> Note that this would likely fail [quest guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md), specifically: _"⚛️ Atomic quests: Per quest, only one thing should need to be answered by the user."_

It already fails this guideline; in fact three things are asked: Can you camp here with a tent? ... with a caravan? ... with a camper van? Ideally these should be split up into 3 separate quests, so that for each the user can answer yes, no or "don't know, hide the quest". For backcountry campsites, only the one about tents could be asked.

In fact the indoor/outdoor seating quest is similar. It happens quite often that I can only answer one of them for sure and am in doubt about the other (for instance because it's winter and there may be outdoor seating in summer, there may be outdoor seating at the back that you can't see from the street, or the place is closed and you can't see if there's indoor seating without going inside).
